digits/templates/models/images/classification/new.html

```diff
@@ -356,16 +356,104 @@ <h4 style="display:inline-block;">Python Layers</h4>
         <div class="col-sm-4">
             <div class="well">
                 <h4>Data Transformations</h4>
+                <div class="form-group{{mark_errors([form.use_mean])}}">
```

Comment:
I feel we haven't done enough of this but you could perhaps move this section to a `data_augmentation.html' template and include it here and in `digits/templates/models/images/generic/new.html`